plugins/inputs/gnmi/gnmi.go

```diff
@@ -533,6 +550,10 @@ const sampleConfig = `
  # prefix = ""
  # target = ""
 
+ ## define webauthtoken
```

Comment:
This is not descriptive enough. Should better be the same as in the readme.

Reply:
Fixed.